runtime/src/test/java/com/redhat/ipaas/runtime/ComponentsITCase.java

```diff
@@ -26,15 +26,15 @@
     @Test
     public void componentsListTest() {
         ListResult<Component> results = restTemplate().getForObject("/api/v1/components", ListResult.class);
-        assertThat(results.getItems()).hasSize(20);
-        assertThat(results.getTotalCount()).isEqualTo(50);
+        assertThat(results.getItems()).hasSize(2);
+        assertThat(results.getTotalCount()).isEqualTo(2);
     }
 
     @Test
     public void componentsGetTest() {
-        Component result = restTemplate().getForObject("/api/v1/components/1", Component.class);
+        Component result = restTemplate().getForObject("/api/v1/components/org.foo:twitter-mention-connector:1.0", Component.class);
```

Comment:
AFAIK there is no concept of Component in the UX. Can you share screens where that applies? As I sent in my email, we have the overloaded terms of Camel Connector and iPaaS Connector which have subtly different meanings sadly, but Component isn't in any design that I'm aware of.